molli/chem/bond.py

```diff
@@ -477,6 +477,10 @@ def get_mol2_type(self) -> str:
 
             case BondType.NotConnected:
                 return MOL2_BOND_TYPE_MAP.inverse[BondType.NotConnected]
+            
+            #Mol2 Format Doesn't Recognize Multi-Attachment Bonds Natively
+            case BondType.Ligand:
```

Comment:
Should we do `case _` here? This will match any bond type, seems like a good fit for mol2.

Reply:
I'm fine with that, would you prefer it separate or remove the bondtype.ligand class?

Reply:
Let's do separate, it will be easier to maintain.

Reply:
Done!